APMrover2/APMrover2.cpp

```diff
@@ -193,7 +193,7 @@ void Rover::gcs_failsafe_check(void)
 void Rover::update_compass(void)
 {
     if (g.compass_enabled && compass.read()) {
-        ahrs.set_compass(&compass);
+        ahrs.update_orientation();
```

Comment:
This is new behaviour, what's the reason behind it? (applies to all vehicles and Replay)

Reply:
set_compass used to do `set_orientation` - which is renamed to `update_orientation`.  So I don't think this is new behaviour.